Raven.Database/Indexing/LuceneQuery.Language.analyzer.lex

```diff
@@ -11,7 +11,7 @@ Digit      [0-9]
 Number     [+-]?{Digit}+
 Decimal    [+-]?{Number}\.{Number}
 EscapeChar \\[^]
-TermStartChar [^ :\t\r\n\f\+\-!\{\}()"^\*\?\\~\[\],]|{EscapeChar}
+TermStartChar [^ :\t\r\n\f\+\-!\{\}()"^\*\?\\~\[\]]|{EscapeChar}
```

Comment:
What happens when a user does a Search on: "Hello! There!" ? 
It should be a single phrase, no?